.github/ISSUE_TEMPLATE/documentation_template.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Documentation request 📖
+description: Suggest new documentation or improvements to existing documentation
+labels: [documentation, untriaged]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit a documentation request!
+  - type: textarea
+    id: suggestion
+    attributes:
+      label: Is there a problem with the current documentation
+      description: A clear and concise description of what the problem is. Ex. I'm unable to find documentation on [...]
```

Comment:
description of the issue.  For instance, "I'm unable to find documentation on [...]"